### ISSUE TITLE
Render office=ngo

### ISSUE THREAD

**Issue by letypequividelespoubelles** [osmandapp/OsmAnd#15889]
Hi,

Could you please render office=NGO for osmand map style? 

Thanks a lot

**Comment by vshcherb** [comment]
I don't think it should be specially rendered probably as just office.

**Comment by letypequividelespoubelles** [comment]
Yes, rendered as an office would be perfect, no need for more specific.

**Comment by letypequividelespoubelles** [comment]
And in fact, maybe all office=* should be rendered?  Maybe some value may have a special icon rendered, but the generic office suits firvtge huge majority of values.
